### PR TITLE
Wy add taxonomy client

### PR DESCRIPTION
**Pull request by wyang-handy**
added a bunch of things in to make a reusable client for service-taxonomy-backend

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,3 +15,7 @@ gem "pry"
 gem 'pry-rescue'
 gem 'pry-stack_explorer'
 gem 'pry-byebug'
+
+group :development, :test do
+  gem 'rspec-rails'
```
Comment (aray-handy) [inline Gemfile]: Not sure if a rails gem needs to be here?

Comment (agulati-handy) [reply Gemfile]: 👍 the gem isn't rails, it's just ruby

File: lib/handy/internal/api/query/taxonomy/taxonomy_service_client.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  module Internal
+    module Api
+      module Query
+        module Taxonomy
+          class TaxonomyServiceClient < Handy::Internal::Api::Client
+            def initialize(url: ENV["TAXONOMY_SERVICE_HOST"], partner_id: ENV["TAXONOMY_SERVICE_PARTNER_ID"], private_key: ENV["API_INTERNAL_PRIVATE_KEY"])
+              super
+            end
+
+            def get_service_categories
```
Comment (aray-handy) [inline lib/handy/internal/api/query/taxonomy/taxonomy_service_client.rb]: This should live in a ServiceCategory class and pass the client in to the constructor.

File: Gemfile.lock [hunk 3]
```diff
@@ -15,7 +15,6 @@ GEM
   specs:
     addressable (2.5.2)
       public_suffix (>= 2.0.2, < 4.0)
-    ast (2.4.0)
```
Comment (aray-handy) [inline Gemfile.lock]: There's no update to the Gemfile - so the Gemfile.lock shouldn't change!

File: lib/handy/internal/api/query/taxonomy.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+require_relative "taxonomy/service_category"
```
Comment (aray-handy) [inline lib/handy/internal/api/query/taxonomy.rb]: Not needed

File: lib/handy/internal/api/query/monolith.rb [hunk 5]
```diff
@@ -8,7 +8,16 @@
 module Handy
```
Comment (aray-handy) [inline lib/handy/internal/api/query/monolith.rb]: We can move `query/monolith` to in here now.

**Pull request closed without merging**